includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -415,22 +416,20 @@ protected function authorize_order_v1( $order_id, $authorize_args = array() ) {
 	 *                                   WP_REST_Response instance.
 	 */
 	public function close_authorization( $request ) {
-		$order_post = $this->is_valid_order( $request['order_id'] );
-		if ( is_wp_error( $order_post ) ) {
-			return $order_post;
+		$or = $this->is_valid_order( $request['order_id'] );
```

Comment:
Can we give this variable a better name, please?

Reply:
done!